google-apis-core/lib/google/apis/core/instrumentation.rb

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'opentelemetry-api'
+
+module Google
+  module Apis
+    module Core
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
```

Comment:
I included the instrumentation base as it surfaces configuration in a consistent way with other instrumented gems.